src/OSPSuite.UI/Binders/ValueOriginBinder.cs

```diff
@@ -143,18 +142,30 @@ private void customDrawCell(object sender, RowCellCustomDrawEventArgs e)
          if (info == null)
             return;
 
-         var firstImage = valueOrigin.Source.Icon.ToImage(IconSizes.Size16x16);
-         var secondImage = valueOrigin.Method.Icon.ToImage(IconSizes.Size16x16);
-         var resultImage = new Bitmap(firstImage.Width + secondImage.Width + IMAGE_OFFSET, firstImage.Height);
+         info.ContextImage = imageFor(valueOrigin);
+         info.CalcViewInfo();
+      }
+
+      private Image imageFor(ValueOrigin valueOrigin)
+      {
+         var sourceImage = valueOrigin.Source.Icon.ToImage(IconSizes.Size16x16);
+         var methodImage = valueOrigin.Method.Icon.ToImage(IconSizes.Size16x16);
```

Comment:
Right now there is always a blank when displaying a method that's not defined and it looks ugly. I am not creating any of that unless required